docs/guides/usage.md

```diff
@@ -0,0 +1,221 @@
+# Execution of pipelines
+
+ZARP consists of three different pipelines. The main pipeline processes the data, the second allows you to download the sequencing libraries from the Sequence Read Archive (SRA), and the third populates a file with the samples and determines sample specific parameters.
+
+If you can create a `samples.tsv` file and fill in the metadata for the different sequencing experiments then the main pipeline can analyze your data.
```

Comment:
I think this should rather go in the section below as it specifically refers to the "main pipeline". Or better yet, just delete it, as the preparation of the sample table is described (somewhat) in step 3.

Reply:
Done